packages/babel-plugin-strip-runtime/src/index.tsx

```diff
@@ -18,9 +21,24 @@ export default declare<PluginPass>((api) => {
     },
     visitor: {
       Program: {
-        exit() {
-          if (this.opts.onFoundStyleRules) {
-            this.opts.onFoundStyleRules(this.styleRules);
+        exit(path) {
+          if (this.opts.styleSheetName) {
+            preserveLeadingComments(path);
+            this.styleRules.forEach((rule) => {
+              // Each found atomic rule will create a new import that uses `@compiled/webpack-loader/css-loader`.
+              // The benefit is two fold:
+              // (1) thread safe collection of styles
+              // (2) caching -- resulting in faster builds (one import per rule!)
+              const params = toURIComponent(rule);
+              path.unshiftContainer(
+                'body',
+                template.ast(
+                  `require("@compiled/webpack-loader/css-loader!@compiled/webpack-loader/css-loader/${this.opts.styleSheetName}.css?style=${params}");`
```

Comment:
can we make this bundler agnostic? The parcel plugins will need to handle this behaviour if the require logic is moved to babel